app/viewmodels/nominees/NomineeStatusHelper.scala

```diff
@@ -111,4 +111,20 @@ object NomineeStatusHelper extends StatusHelper {
       case _ => false
     }
   }
+
+  def validateDataFromOldService(userAnswers: UserAnswers): Boolean = {
+    (userAnswers.get(IsAuthoriseNomineePage), userAnswers.get(ChooseNomineePage)) match {
+      case(Some(true), Some(true)) =>
+        userAnswers.get(IndividualNomineeNamePage) match {
+          case Some(name) if name.title == SelectTitle.UnsupportedTitle => false
+          case _ => true
+        }
+      case(Some(true), Some(false)) =>
+        (userAnswers.get(OrganisationAuthorisedPersonNamePage), userAnswers.get(OrganisationNomineeContactDetailsPage)) match {
+          case (Some(name), Some(contact)) => name.title != SelectTitle.UnsupportedTitle || contact.email!=""
```

Comment:
this should be && - if email is empty but title is unsupported, this will return true. also, instead of `!=""` you can use `.nonEmpty` I believe